### DISCUSSION TITLE
windows 7 nodejs v5

### DISCUSSION THREAD

**Discussion by Toodo** [SudhanPlayz/Discord-MusicBot#1071] [Q&A]
Hello i have the bot on replit now but replit just rate limited after an hr or two and i get this errores  (https://github.com/SudhanPlayz/Discord-MusicBot/issues/1061)
so i trying to install the bot on my old computer with win 7. but nodejs does not support win 7 so is there a workaround to install nodejs 16 on win 7?
i have tried this:

i install v13.6 then download nodejs binary 16 paste files in the folder but only got node to worked but not npm.

and i also try to block platform check in systempropertiesadvanced with "NODE_PATH, value "C:\nodejs64\node_modules " and NODE_SKIP_PLATFORM_CHECK, value "1"

**Comment by JotaroKujo0525** [comment]
upgrade your win version?

**Reply by JotaroKujo0525** [reply]
kind of a logical answer tho, if your computer is windows 7, then just upgrade it. there's no workarounds in this since nodejs is really not supported on windows 7
